consensus/core/src/metrics.rs

```diff
@@ -91,14 +92,21 @@ impl NodeMetrics {
                 "The current threshold clock round. We only advance to a new round when a quorum of parents have been synced.",
                 registry,
             ).unwrap(),
+            invalid_blocks: register_int_counter_vec_with_registry!(
+                "invalid_blocks",
+                "Number of invalid blocks per peer authority",
+                &["authority"],
```

Comment:
I thought about it and I'm a bit worried about cardinality, since this metric is per-authority. Error details are logged at info level in `AuthorityService::handle_send_block()` right now.

Reply:
I see the concern. I believe we already have metrics with even higher cardinality - ex the channel metrics, code scope etc - not that this is an argument on its own, but I mean I see this as manageable. Also I don't see those 
 errors happening frequently - I mean if they do happen then our network is in a byzantine position and we might want to prioritise here early detection. We can think about it and add in a later iteration if indeed needed.

Reply:
Channel and scope metrics don't have a peer or authority label. When emitted from a validator, each of them has 20~30 different streams. With a peer label, the metric already has > 100 streams, and after multiplied with the cardinality of status, there can be thousands. This metric should not have high cardinality under normal operations though. Let me come back to this later.